README.md

```diff
@@ -237,20 +237,10 @@ $A.enqueueAction(action);
 
 Both methods shown above will print out the same information.
 
-## Supported Services
+## WebSocket Support
+Both the Text to Speech and Speech to Text services have API endpoints which support the use of WebSockets. However, the use of WebSockets are not supported natively by Apex. If you would like to take advantage of this functionality, the recommendation would be to leverage the [Watson Node SDK](https://github.com/watson-developer-cloud/node-sdk).
```

Comment:
Use https://github.com/watson-developer-cloud/speech-javascript-sdk not node-sdk